plugins/webhooks/index.js

```diff
@@ -22,17 +64,17 @@ const DEFAULT_MAX_BYTES = 1048576;
  * @param {String} pipelineBranch  The pipeline branch
```

Comment:
need to add new param jsdoc

Reply:
Thank you for review. 
I add param in jsdoc.